onboarding/source/amb_light_service.c

```diff
@@ -17,12 +26,23 @@
 
 /* USER CODE BEGIN */
 // Define light service queue config here
-
+#define LIGHT_QUEUE_LENGTH 10
+#define LIGHT_QUEUE_SIZE sizeof(light_event_t)
```

Comment:
It'd be clearer to name this `LIGHT_QUEUE_ITEM_SIZE` since it's just the size of a single item in the queue